resources/waf_rules.go

```diff
@@ -0,0 +1,64 @@
+package resources
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/waf"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func WafRules() *schema.Table {
```

Comment:
You are missing [tags ](https://docs.aws.amazon.com/waf/latest/APIReference/API_waf_ListTagsForResource.html)for WafRules, WebAcls and WafRuleGroups.

Reply:
@roneli Thanks! I was able to add tags for WebACLs but I was unable to get the tags for WafRules and WafRuleGroups because both resources don't return a resource arn and I wasn't able to get it at all.

Reply:
After searching through the terraform aws code I was able to find a solution. Apparently it is possible to generate the ARN dynamically: 
https://github.com/hashicorp/terraform-provider-aws/blob/f4edafb9fed9d7112e6373adcbca152804213693/aws/resource_aws_waf_rule.go#L160-L166

Reply:
If we are generating this column, it might be beneficial to add this a column to the Table as well.

Reply:
Could you elaborate a little bit on what you mean with that? The `tags` column is defined in the `schema.Column` slice which should add it to the table, right?

Reply:
I meant the "arn" column